src/components/todos/todos.container.js

```diff
@@ -3,6 +3,6 @@ import Component from './todos'
 
 export default inject(store => ({
   todos: store.data.todos[store.ui.footer.get().filter].getKeys(),
-  allCompleted: store.data.todos.all.getLength() === store.data.todos.completed.getLength(),
+  allCompleted: store.ui.footer.todosLeft === 0,
```

Comment:
This is not part of the solution.
But, by deleting `all` store it crashes here. So I did a quick fix :)